hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -135,6 +136,10 @@
    * Quota of key count allocated for the bucket.
    */
   private long quotaInNamespace;
+  /**
+   * Bucket Type.
+   */
+  private BucketType bucketType;
```

Comment:
I think we need a default bucketType here otherwise Line586 can trigger NPE.

Reply:
I'll add a default bucket type.